a10_octavia/controller/worker/flows/a10_member_flows.py

```diff
@@ -779,3 +779,240 @@ def get_create_member_snat_pool_subflow(self):
+            if CONF.a10_global.network_type == 'vlan':
+                batch_update_members_flow.add(
+                    vthunder_tasks.DeleteInterfaceTagIfNotInUseForMember(
+                        name='delete_unused_interface_tag_in_member_' + m.id,
+                        inject={constants.MEMBER: m},
+                        requires=[
+                            constants.MEMBER,
+                            a10constants.VTHUNDER]))
+            batch_update_members_flow.add(database_tasks.DeleteMemberInDB(
+                inject={constants.MEMBER: m},
+                name='{flow}-{id}'.format(
+                    id=m.id, flow=constants.DELETE_MEMBER_INDB)))
+            batch_update_members_flow.add(database_tasks.DecrementMemberQuota(
+                inject={constants.MEMBER: m},
+                name='{flow}-{id}'.format(
+                    id=m.id, flow=constants.DECREMENT_MEMBER_QUOTA_FLOW)))
+        batch_update_members_flow.add(
+            self.get_delete_member_vrid_internal_subflow(constants.POOL, old_members))
+
+        # for creation of members
+        batch_update_members_flow.add(lifecycle_tasks.MembersToErrorOnRevertTask(
+            name='{flow}-created'.format(
+                flow=constants.MEMBER_TO_ERROR_ON_REVERT_FLOW),
+            inject={constants.MEMBERS: new_members}))
+        for m in new_members:
+            batch_update_members_flow.add(database_tasks.MarkMemberPendingCreateInDB(
+                name='mark-member-pending-create-in-db-' + m.id,
+                inject={constants.MEMBER: m}))
+            batch_update_members_flow.add(a10_database_tasks.CountMembersWithIP(
+                name='count-member-with-ip-' + m.id,
+                inject={constants.MEMBER: m},
+                provides=a10constants.MEMBER_COUNT_IP))
+            batch_update_members_flow.add(self.get_batch_update_member_snat_pool_subflow(m))
+            batch_update_members_flow.add(server_tasks.MemberCreate(
+                name='member-create-' + m.id,
+                inject={constants.MEMBER: m},
+                requires=(constants.MEMBER, a10constants.VTHUNDER, constants.POOL,
+                          a10constants.MEMBER_COUNT_IP, constants.FLAVOR)))
+            batch_update_members_flow.add(database_tasks.MarkMemberActiveInDB(
+                inject={constants.MEMBER: m},
+                name='{flow}-{id}'.format(
+                    id=m.id, flow=constants.MARK_MEMBER_ACTIVE_INDB)))
+
+        # for updating of members
+        batch_update_members_flow.add(
+            lifecycle_tasks.MembersToErrorOnRevertTask(
+                name='{flow}-updated'.format(
+                    flow=constants.MEMBER_TO_ERROR_ON_REVERT_FLOW),
+                # updated_members is a list of (obj, dict), only pass `obj`
+                inject={constants.MEMBERS: [m[0] for m in updated_members]}))
+        for m, um in updated_members:
+            um.pop('id', None)
```

Comment:
Default return would be None anyway and we aren't returning anything

Reply:
If we don't add `um.pop('id', None)` here, then we will get below error for member batch update:

`Sep 22 09:41:43 openstack-2 a10-octavia-worker[13196]: ERROR oslo_messaging.rpc.server   File "/opt/stack/octavia/octavia/controller/worker/tasks/database_tasks.py", line 1390, in execute
Sep 22 09:41:43 openstack-2 a10-octavia-worker[13196]: ERROR oslo_messaging.rpc.server     **update_dict)
Sep 22 09:41:43 openstack-2 a10-octavia-worker[13196]: ERROR oslo_messaging.rpc.server TypeError: update() got multiple values for keyword argument 'id'`